solutions/usaco-515.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: usaco-515
+source: USACO Gold 2015 January
+title: Moovie Mooving
+author: Melody Yu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_movie_gold.html)
+
+## Video Solution
+
+Note: The video solution might not be the same as other solutions. Code in C++.
+<Youtube id="oKm_GZD9u1E" />
+
+**Time Complexity:** $\mathcal{O}(2^N+CN)N$
```

Comment:
> **Time Complexity:** $\mathcal{O}(2^N+CN)N$

?? check the latex

Reply:
I checked in usaco.guide/editor and it shows O((2^N+CN)N) ... anything wrong ?

Reply:
the code above seems to be missing parentheses